documentation/DotNetCli.md

```diff
@@ -0,0 +1,34 @@
+
+# Using StyleCop Analyzers with dotnet cli
+
+StyleCop Analyzers can be used in dotnet cli projects, including asp.net core.
+The tooling support is currently not great and the analyzers only run when the project is compiled and there is currently no way to invoke the code fixes. What does work however is running Stylecop Analyzers in ubuntu on coreclr and OSX (probably) works too.
+To set this up a few steps are required:
+
+First the StyleCopAnalyzers nuget package has to be added to the project. Optimally the package should be marked as `build` type so it is not included as a package by other projects consuming it.
+For this add the following to the dependencies section of the project.json file:
+```json
+    "StyleCop.Analyzers": {
+      "version": "1.0.0",
+      "type": "build"
+    }
+```
+
+If the project is restored and built right now this will already run the analyzers. A few extra steps are needed if they should be configured.
+
+## Rulesets and stylecop.json
+
+To supply a ruleset file and a stylecop.json configuration file to the compiler they have to be manually added as arguments to the compiler. For this add the following under the `compilationOptions` node in the project.json file:
+```json
+    "additionalArguments": [ "/ruleset:path/to/ruleset.ruleset", "/additionalfile:path/to/stylecop.json" ]
+```
+
+## Enabling xml documentation processing
```

Comment:
:bulb: Consider merging this section with the documentation for [SA0001](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA0001.md).